frameworks/opennmt_py/requirements.txt

```diff
@@ -1,9 +1,7 @@
 boto3
+flask
```

Comment:
probably not wanted?

Reply:
The OpenNMT-py server is using flask, I'm forwarding the requests to it for now.